pkg/ingester/ingester_v2.go

```diff
@@ -1386,7 +1418,7 @@ func (i *Ingester) getTSDBUsers() []string {
 	return ids
 }
 
-func (i *Ingester) getOrCreateTSDB(userID string, force bool) (*userTSDB, error) {
+func (i *Ingester) getOrCreateTSDB(userID string, force bool, gl *GlobalLimits) (*userTSDB, error) {
```

Comment:
Why do we need to pick it in input, instead of calling `i.getGlobalLimits()`?

Reply:
We don't. Fixed.